spec/models/labware_creators/quadrant_split_plate_spec.rb

```diff
@@ -207,7 +180,24 @@
         )
       end
 
+      before do
+        allow(SearchHelper).to receive(:merger_plate_names).and_return(stock_purpose_name)
+
+        stub_v2_user(user)
+
+        stub_v2_labware(stock_plate_v2)
```

Comment:
Is there a nice way of handling this where we can give it a list of plates instead of invoking it for each plate.

Reply:
We could add more stub methods, but there are already a lot of them.  We could use some sort of nasty reflection method where we tell it to stub resources and pass the type as a string, constantizing it to get a reference to the individual stub method and looping inside the stubs module.  Or we could set this up something like:

```
[child_plate_a, child_plate_b, child_plate_c, child_plate_d, child_plate_e].each { |labware| stub_v2_labware(labware) }
```

But as these are tests, do not need to conform to DRY approaches, and I want the setup to be very clear to anyone who needs to understand what the setup and outcomes are, I don't think we should do any of that.